sdk/appconfiguration/azure-spring-cloud-appconfiguration-config/src/main/java/com/microsoft/azure/spring/cloud/config/properties/ConfigStore.java

```diff
@@ -35,7 +35,7 @@
 
     private boolean failFast = true;
 
-    private boolean useFeatureManagement = false;
+    private FeatureFlagStore featureFlags = new FeatureFlagStore();
```

Comment:
make it final?

Reply:
@stliu I don't think it can be because FeatureFlagStore, is itself a set of configuration properties. Lines 106-108 would need to be removed.